devicetypes/aeotec/aeotec-aerq-sensor.src/aeotec-aerq-sensor.groovy

```diff
@@ -0,0 +1,372 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Z-Wave Temp/Humidity Sensor
+ *
+ *  Author: Chris
+ *  Date: 2021-01-29
+ */
+
+metadata {
```

Comment:
Since You have already created a new pull request for those devices
https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/56937
should this one be closed ?

Reply:
@KKlimczukS, i thought i removed my request from SmartThings Developers page, please close #56936 and keep #56937 open (it has minor text changes that were wrong in this pull request).